src/style.css

```
@@ -1,3 +1,73 @@
-body {
+/* reset browser default margins */
+* {
   margin: 0;
+  padding: 0;
+}
+
+html {
+  font-family: Roboto, serif;
+  font-weight: 500;
+  font-size: 12px;
+
+  --blue: #00ACDC;
+  --header-heigth: 60px;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  text-transform: uppercase;
+  height: var(--header-heigth);
+  padding: 0 50px;
+}
+
+.logo{
+  display: flex;
+  align-items: center;
+}
+
+.tab-container {
+  display:flex;
+  flex: row;
+  height:100%;
+}
+
+.tab {
+  position:relative;
+  height:100%;
+  margin-left: 20px;
+  display:flex;
+  align-items:center;
+  justify-content: space-around;
+}
+
+.tab:first-child {
+  margin:0;
+}
+
+.nav-link {
+  display: flex;
+  height: 100%;
+  align-items: center;
+  color:black;
+  text-decoration: none;
+}
+
+.nav-link:visited {
+  color:black;
+}
+
+.nav-link:hover,  .is-active, .is-active:visited{
```

```suggestion
.nav-link:hover, 
.is-active,
.is-active:visited {
```
Don't put all selectors in one line